README.md

```diff
@@ -40,6 +40,8 @@ ALTER USER postgres WITH SUPERUSER;
 \q
 ```
 
+If Postgres already has a user not called `postgres`, you can create one by running `psql -U postgres`
```

Comment:
This is literally wrong. `-U` provides the user to login to. Since the `postgres` is generally already created by default, `psql -U postgres` generally works. Please read documentation and fix accordingly.

Reply:
Resolved in the following commit